components/graphs/Graph.tsx

```diff
@@ -23,6 +24,17 @@ const Graph: FC<{ siteUUID: string }> = ({ siteUUID }) => {
       )
     : 0;
 
+  const clearSkyEstimateTrimmed = clearskyData?.clearsky_estimate.filter(
+    (clearSkyDataPoint: ClearSkyDataPoint) => {
+      return (
+        (forecastData?.forecast_values[0]?.target_datetime_utc ?? 0) <=
+          clearSkyDataPoint.target_datetime_utc &&
+        (forecastData?.forecast_values[forecastData.forecast_values.length - 1]
+          ?.target_datetime_utc ?? 0) >= clearSkyDataPoint.target_datetime_utc
+      );
+    }
+  );
```

Comment:
I like this, but since Neha's [forecastDataOverRange](https://github.com/openclimatefix/pv-sites-mobile/blob/7fae75cdc065685bfb7b86856091995378442aae/lib/graphs.ts#L86) function is now merged in, can it be used instead? Lmk if not